goto.py

```diff
@@ -34,6 +34,8 @@ def __init__(self):
             self.have_argument = dis.HAVE_ARGUMENT
             self.jump_unit = 1
 
+        self.has_loop_blocks = 'SETUP_LOOP' in dis.opmap
```

Comment:
I'm wondering if we should rather check for `'SETUP_LOOP' in dis.opmap` in place than aliasing the result of this check here? It doesn't seem to save much writing, neither does it seem to improve performance.

Reply:
I think it's cleaner, and it's far from a sole occurrence - see the later PRs which add more such checks, some (I think) used more than once.